websites/P/Puray Animes/presence.ts

```diff
@@ -0,0 +1,54 @@
+const presence = new Presence({
+		clientId: "972246349917610054"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+presence.on("UpdateData", () => {
+	const { pathname } = window.location,
+		data: PresenceData = {
```

Comment:
for the sake of consistency, please
```suggestion
		presenceData: PresenceData = {
```

Reply:
Please stop resolving this without addressing the review, you need to rename every instance of the variable not just this one

Reply:
I changed the variable name.